packages/core/src/utils/__tests__/security.spec.ts

```diff
@@ -329,4 +329,65 @@ describe('validateSecurity', () => {
       });
     });
   });
+
+  describe('AND relation between security schemes', () => {
+    describe('when given only one key', () => {
+      test.only('fails with an invalid security scheme error', () => {
+        expect(
+          validateSecurity<any, any>(
+            { headers: { 'x-api-key': 'abc123' } },
+            {
+              security: [
```

Comment:
This could use the same `const securtyScheme` as above, using contexts to reduce duplication! 🙌